CPRynner/CPRynner.py

```diff
@@ -136,72 +120,66 @@ def __init__(self, cluster_address, tasks_per_node, work_dir, setup_script):
         main_sizer.Add(work_dir_sizer, 0, wx.ALL, 5)
         main_sizer.Add(setup_script_label_sizer, 0, wx.ALL, 5)
         main_sizer.Add(setup_script_field_sizer, 0, wx.ALL, 5)
+        main_sizer.Add(run_command_label_sizer, 0, wx.ALL, 5)
+        main_sizer.Add(run_command_field_sizer, 0, wx.ALL, 5)
         main_sizer.Add(button_sizer, 0, wx.ALL | wx.ALIGN_CENTER, 5)
-
+ 
         self.panel.SetSizer(main_sizer)
 
 
 class LoginDialog(wx.Dialog):
     """
     A dialog window asking for a username and a password
     """
-
-    def __init__(self, username=''):
+ 
+    def __init__(self, username = ''):
```

Comment:
Avoiding spaces around `=` in the context of function parameters and arguments is the generally preferred style in Python; I would revert this change.